### ISSUE TITLE
[Bug][A11Y][Radio Button] Keyboard focus is not visible on Android

### ISSUE THREAD

**Issue by pya35** [Orange-OpenSource/ouds-flutter#857]
### Prerequisites

- [x] I have [searched the backlog](https://github.com/Orange-OpenSource/ouds-flutter/issues) for duplicate or closed feature requests

### Your test device

S25

### OS Version

Android 16

### App version

2.0.0 (710)

### Describe the issue

Keyboard focus is not visible on radio button on Android, not reproduced on iOS

### Expected Behavior

Focus shoudl be visible
